drivers/video/video_stm32_dcmi.c

```diff
@@ -174,6 +281,20 @@ static int stm32_dma_init(const struct device *dev)
 		return -EIO;
 	}
 
+#if defined(CONFIG_SOC_SERIES_STM32U5X)
+	if (HAL_DMAEx_List_Init(&hdma) != HAL_OK) {
+		LOG_ERR("HAL_DMAEx_List_Init Failed");
+		return -EINVAL;
+	}
+	if (stm32_dma_list_init(&hdma) != HAL_OK) {
+		LOG_ERR("stm32_dma_list_init Failed");
+		return -EINVAL;
+	}
+	if (HAL_DMA_ConfigChannelAttributes(&hdma, DMA_CHANNEL_NPRIV) != HAL_OK) {
+		LOG_ERR("HAL_DMA_ConfigChannelAttributes Failed");
+		return -EINVAL;
+	}
```

Comment:
Just noticing the different style through the source, in case you were interested in smoothing it up:
- above uses `err = HAL_...(); if (err != HAL_OK) {}`
- here it uses `if (HAL_...() != HAL_OK) {}`